components/bl-autocomplete-component/src/text-field/index.js

```diff
@@ -25,7 +23,7 @@ export const TextField = forwardRef((props, ref) => {
   const { onButtonClearClick, onChange } = eventHandlers;
 
   const inputRef = useRef(null);
-  const autocompleteId = useMemo(() => generateId(), []);
+  const autocompleteId = useMemo(() => BackendlessUI.UUID.short(), []);
```

Comment:
Does React.useId() works here? Not critical, just for information.

Reply:
React.useId() doesn't work here because it appeared in React 18. And we use React 17.